lib/secure_headers/configuration.rb

```diff
@@ -14,9 +15,21 @@ class << self
       #
       # Returns the newly created config.
       def default(&block)
-        config = new(&block)
-        add_noop_configuration
-        add_configuration(DEFAULT_CONFIG, config)
+        if defined?(@default_config)
```

Comment:
In the spirit of preserving the integrity of the default config I added this. It is my assumption that a caller should only ever being calling this configuration block a single time, correct? If it is called more than once it implies that we could be running into strange runtime behavior, since the call order could go something like:

``` ruby
Configuration.default
...
some runtime override
...
Configuration.default
```

Reply:
> It is my assumption that a caller should only ever being calling this configuration block a single time, correct?

There were guards in place to prevent doing exactly that. Not sure if they survived the change.

Reply:
But don't the tests do a bunch of `Configuration.default {...}` between tests? That implies it was ok to reconfigure the default config a bunch of times, no?